### ISSUE TITLE
Add detail to implicit default realm evaluation failure.

### ISSUE THREAD

**Issue by Denney-tech** [jborean93/PSOpenAD#8]
I would very much like to use local default realm and implicit domain controller and kerberos based authentication without having to create a session.  

I can create a session and test just fine like so:
``` powershell
$session = New-OpenADSession -ComputerName foo-dca.contoso.com -AuthType Kerberos -UseSSL
Get-OpenADWhoami -Session $session
```
`u:CONTOSO\bar`

But without specifying a session, I get a very unhelpful error:
``` powershell
Get-OpenADWhoami
```
`Get-OpenADWhoami: Cannot determine default realm for implicit domain controller.`

Looking at your [about_OpenADSessions](https://github.com/jborean93/PSOpenAD/blob/main/docs/en-US/about_OpenADSessions.md), the default_realm and implicit domain controller should be found as long as /etc/krb5.conf is configured correctly and `nslookup -type=SRV _ldap._tcp.dc._msdcs.contoso.com` returns results. Alternatively to the krb5.conf file having the default_realm set, the `hostname` domain suffix could be used. 

I have confirmed that default_realm is set in /etc/krb5.conf (and is readable by my user).
I have confirmed that `nslookup -type=SRV _ldap._tcp.dc._msdcs.contoso.com` returns all 4 expected dc's (in my case).
While it was not originally set, I have gone ahead and configured the local hostname to include its domain name.

Despite all requirements supposedly being met, PSOpenAD is not able to determine and use the default_realm or implicit domain controllers. It would be nice therefore to have more explicit messages to _why_ PSOpenAD "Cannot determine default realm for implicit domain controller."

**Comment by jborean93** [comment]
This is a good idea as right now it is hard to determine why the implicit realm/dc failed. In saying that I believe this has been fixed with https://github.com/jborean93/PSOpenAD/pull/5, specifically the changes at https://github.com/jborean93/PSOpenAD/pull/5/files#diff-f36da8d3ddd75d05d0b813f67804c2469b23819be3ee2d3911523435f515deadR139-R140. I originally had the `libkrb5` use an so without a major version which worked on my dev host due to changes I've made there but it wasn't generic enough. I'll try and push out the new preview sometime shortly which includes that fix.
